File: _persist/temp/venda/CSharp/Produto.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("PRODUTO")]
    public class Produto : CollectionItem
    {
        [Campo("ID_PRODUTO", CampoTipo.tfKey)]
        public string Id_Produto { get; set; }
        [Campo("U_VERSION", CampoTipo.tfNul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.tfReq)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.tfReq)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("CD_PRODUTO", CampoTipo.tfReq)]
        public int Cd_Produto { get; set; }
        [Campo("DS_PRODUTO", CampoTipo.tfReq)]
        public string Ds_Produto { get; set; }
        [Campo("CD_ESPECIE", CampoTipo.tfReq)]
        public string Cd_Especie { get; set; }
        [Campo("CD_NCM", CampoTipo.tfReq)]
        public string Cd_Ncm { get; set; }
        [Campo("CD_CST", CampoTipo.tfReq)]
        public string Cd_Cst { get; set; }
        [Campo("CD_CSOSN", CampoTipo.tfReq)]
        public string Cd_Csosn { get; set; }
        [Campo("PR_ALIQUOTA", CampoTipo.tfReq)]
        public double Pr_Aliquota { get; set; }
        [Campo("TP_PRODUCAO", CampoTipo.tfReq)]
        public int Tp_Producao { get; set; }
        [Campo("VL_CUSTO", CampoTipo.tfReq)]
        public double Vl_Custo { get; set; }
        [Campo("VL_VENDA", CampoTipo.tfReq)]
        public double Vl_Venda { get; set; }
        [Campo("VL_PROMOCAO", CampoTipo.tfReq)]
        public double Vl_Promocao { get; set; }
    }
}

File: _persist/temp/venda/Java/Produto.java
package MORM.Java.Models;

import java.util.Date;
import MORM.Java.Classes;

@Tabela("PRODUTO")
public class Produto extends CollectionItem
{
    @Campo("ID_PRODUTO", CampoTipo.tfKey)
    public String Id_Produto;
    @Campo("U_VERSION", CampoTipo.tfNul)
    public String U_Version;
    @Campo("CD_OPERADOR", CampoTipo.tfReq)
    public Integer Cd_Operador;
    @Campo("DT_CADASTRO", CampoTipo.tfReq)
    public DateTime Dt_Cadastro;
    @Campo("CD_PRODUTO", CampoTipo.tfReq)
    public Integer Cd_Produto;
    @Campo("DS_PRODUTO", CampoTipo.tfReq)
    public String Ds_Produto;
    @Campo("CD_ESPECIE", CampoTipo.tfReq)
    public String Cd_Especie;
    @Campo("CD_NCM", CampoTipo.tfReq)
    public String Cd_Ncm;
    @Campo("CD_CST", CampoTipo.tfReq)
    public String Cd_Cst;
    @Campo("CD_CSOSN", CampoTipo.tfReq)
    public String Cd_Csosn;
    @Campo("PR_ALIQUOTA", CampoTipo.tfReq)
    public Float Pr_Aliquota;
    @Campo("TP_PRODUCAO", CampoTipo.tfReq)
    public Integer Tp_Producao;
    @Campo("VL_CUSTO", CampoTipo.tfReq)
    public Float Vl_Custo;
    @Campo("VL_VENDA", CampoTipo.tfReq)
    public Float Vl_Venda;
    @Campo("VL_PROMOCAO", CampoTipo.tfReq)
    public Float Vl_Promocao;
}